scapy/contrib/mpls.py

```diff
@@ -16,30 +16,47 @@
 # scapy.contrib.status = loads
 
 from scapy.packet import Packet, bind_layers, Padding
-from scapy.fields import BitField,ByteField
+from scapy.fields import BitField,ByteField,ShortField
 from scapy.layers.inet import IP
 from scapy.layers.inet6 import IPv6
 from scapy.layers.l2 import Ether, GRE
 from scapy.compat import orb
 
+class EoMCW(Packet):
+    name = "EoMCW"
+    fields_desc = [ BitField("zero", 0, 4),
+                    BitField("reserved", 0, 12),
+                    ShortField("seq", 0) ]
+    def guess_payload_class(self, payload):
+        if len(payload) >= 1:
+            return Ether
+        return Padding
+
 class MPLS(Packet):
-   name = "MPLS"
-   fields_desc =  [ BitField("label", 3, 20),
```

Comment:
Please do not change code when not needed, it breaks the history.

Reply:
The indentation is weird, first indentation 3 spaces, subsequent 4 spaces.  My editor automatically fixed it.
Do you not want the indentation fixed?